src/collections/_documentation/workflow/user-settings.md

```diff
@@ -0,0 +1,187 @@
+
+{% asset user-settings/us-shistory.png alt="Session History tab in Security under User Settings" width="380" %}
+
+# Notifications
+
+Sentry notifications are broadly categorized into alerts and non-alert notifications. Alerts can be sent to many supported [integrations]({%- link _documentation/workflow/integrations/index.md -%}#webhook-alerts). Non-alert notifications only go to email.
+
+## Alerts & Notifications
+
+You can define Project Alerts in  [Project] > Project Settings > Alerts. Slide the toggle under Send Me Alerts to receive team alerts. You can also fine-tune the alerts you receive per project.
+
+Control workflow notifications (for example, in issue assignment, resolution status, and comments) by clicking a radio button under Send Me Workflow Notifications. You can also fine-tune workflow notifications per project.
+
+For more details, see the [full documentation on Alerts & Notifications]({%- link _documentation/workflow/alerts-notifications/index.md -%})
+
+## Email Routing
+
+Route email notifications to an alternative email address and keep your project notifications organized. 
+
+{% asset user-settings/us-eroute.png alt="Email Routing field in Notifications under User Settings" width="450" %}
+
+## Weekly Reports
+
+Sentry generates reports per project and sends them once a week. You can fine-tune your reports per project.
+
+{% asset user-settings/us-wrfield.png alt="Weekly Reports field in Notifications under User Settings" width="450" %}
+
+The following report sample shows details on Events Seen This Week, Events by Issue Type, Events by Project, and activity over time.
+
+{% asset user-settings/us-ereport.png alt="Weekly Report email sample" width="745" %}
+
+## Deploy Notifications
+
+Choose a Deploy Notifications option to keep track of deployment details like release version, environments, and commit reviews. You can fine-tune options from the dropdown next to each organization.
+
+{% asset user-settings/us-deploy.png alt="Deploy Notifications selection in Notifications under User Settings" width="673" %}
+
+## My Activity
+
+Updates regarding your user account actions, such as claiming unassigned issues, can be turned into an email notification in this section.
+
+{% asset user-settings/us-myact.png alt="My Activity selection in Notifications under User Settings" width="450" %}
+
+{% capture __alert_content -%}
+You can find a shortcut to the Emails panel at the bottom of this section.
+
+{% asset user-settings/us-npanellink.png alt="A link to the Email panel in Notifications under My Activity" width="465" %}
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+%}
+
+## Emails
+
+The email address used to login to your Sentry account is by default your primary email address. You can add an alternative email address under the Add Secondary Emails section.
```

Comment:
Where possible, let's get rid of "You can" and just go straight to the verb "Add alternate email addresses using the "Add Secondary Emails" section."